packages/frontend/src/components/common/balance/BalanceDisplay.js

```diff
@@ -92,6 +96,8 @@ const BalanceDisplay = ({
         }
     };
 
+    const markIfNotNearAndNotWhitelisted = !isNear && !isWhiteListed ? '#FF585D' : '';
```

Comment:
are there any constants for colors?

Reply:
There isn't a constant for this color. Hex value used through the code instead

Reply:
we are using this one more than one, should be added somewhere?